docs/app/Examples/elements/Segment/Groups/SegmentNestedSegmentsExample.js

```diff
@@ -0,0 +1,23 @@
+import React, {Component} from 'react';
+import {Segment, Segments} from 'stardust';
+
+export default class SegmentNestedSegmentsExample extends Component {
+  render() {
+    return (
+      <Segments>
+        <Segment>Top</Segment>
+        <Segments>
+          <Segment>Nested Top</Segment>
+          <Segment>Nested Middle</Segment>
+          <Segment>Nested Bottom</Segment>
+        </Segments>
+        <Segments className='horizontal'>
+          <Segment>Top</Segment>
+          <Segment>Middle</Segment>
+          <Segment>Bottom</Segment>
+        </Segments>
+        <Segment>Bottom</Segment>
+      </Segments>
+    );
+  }
+}
```

Comment:
Yeah that's what I was wondering. The docs examples definitely have situations that rarely happen in the wild, so this might not be a big issue.